cucumber-messages/go/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/cucumber/cucumber-messages-go/v5
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
+	github.com/golang/protobuf v1.3.2 // indirect
```

Comment:
Hmm I wonder where this comes from. We're using `gogo/protobuf`, not `golang/protobuf`...